README.md

```diff
@@ -2,78 +2,16 @@
 SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 -->
 
-# beman.example: A Beman Library Example
+# beman.dump: Implementation of proposed `std::dump`
```

Comment:
```suggestion
# beman.dump: A tool for dumping an object's value to standard output
```

IMO it's better, from a user perspective, to describe the functionality this contains instead of forcing someone to look elsewhere.